controllers/tiflowcluster_controller.go

```diff
@@ -68,6 +76,8 @@ func NewTiflowClusterReconciler(cli client.Client, clientSet kubernetes.Interfac
 func (r *TiflowClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = log.FromContext(ctx)
 
+	klog.Infof("Current Version: %s", Version)
```

Comment:
What's this used for?

Reply:
To identify the test versions at different times, the production environment will remove

Reply:
Then don't merge them in this PR? I'm afraid we forget to remove them later.

Reply:
ok